webtau-junit5-examples/src/test/java/com/example/tests/junit5/CustomerCrudSeparatedJavaTest.java

```diff
@@ -1,61 +1,41 @@
-/*
- * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.example.tests.rest;
+package com.example.tests.junit5;
```

Comment:
Why did you delete the copyright notice?

Reply:
it is being pulled into a doc as an example